src/utils/report.js

```diff
@@ -57,6 +57,7 @@ export default function ({
 
   const severity = get(result.stylelint, [ "ruleSeverities", ruleName ], "ignore")
 
+  /* istanbul ignore if */
   if (typeof severity === "undefined") {
```

Comment:
@davidtheclark `severity` will never `"undefined"`, because we return "ignore" if it is undefined, bug maybe happened if `lodash` not return `"ignore"`, but it is bug "lodash"